packages/testlab/src/validate-api-spec.ts

```diff
@@ -3,20 +3,15 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import * as SwaggerParser from 'swagger-parser';
-import {OpenApiSpec} from '@loopback/openapi-spec';
+import {OpenApiSpec} from '@loopback/openapi-spec-types';
+const validator = require('swagger2openapi/validate.js');
+import * as util from 'util';
+const promisify = util.promisify;
```

Comment:
Nitpick, feel free to ignore:

These two lines can be simplified as follows - I think we are already using the shorter variant everywhere else:

```ts
import {promisify} from 'util';
```

Reply:
good catch. The old way of importing `promisify` is more complicated, I didn't merge its import to `util` after updating to the latest approach.